<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>103</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>103</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxNetwork">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>290</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string> Network </string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>258</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelIP">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="maximum">
        <number>65000</number>
       </property>
       <property name="value">
        <number>5660</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCOM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>290</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>COM</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>260</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayoutCOM">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDeviceName">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Device name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditDeviceName">
       <property name="text">
        <string>/dev/ttyUSB1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBoxNetwork</sender>
   <signal>clicked(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>groupboxNetworkClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxCOM</sender>
   <signal>clicked(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>groupBoxCOMClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>pressed()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>btnOKClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>pressed()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>btnCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>groupBoxNetworkClicked(bool)</slot>
  <slot>groupBoxCOMClicked(bool)</slot>
  <slot>btnOKClicked()</slot>
  <slot>btnCancelClicked()</slot>
 </slots>
</ui>
